### PR TITLE
Address #13894 nits

### PR DESCRIPTION
**Pull request by jibi**
Just addressing a couple of nits which slipped through https://github.com/cilium/cilium/pull/13894

### CODE REVIEW

**Review by twpayne — APPROVED** [review]

**Review by rolinh — APPROVED** [review]
Thanks!

File: daemon/cmd/daemon_main.go [hunk 1]
```diff
@@ -849,7 +849,7 @@ func init() {
 	flags.MarkDeprecated(option.HubbleFlowBufferSize, fmt.Sprintf("Use %s instead.", option.HubbleEventBufferCapacity))
 	option.BindEnv(option.HubbleFlowBufferSize)
 
-	flags.Int(option.HubbleEventBufferCapacity, observeroption.Default.MaxFlows.AsInt(), "Capacity of Hubble events buffer. The provided value must be on less than an integer power of two and no larger than 65535 (ie: 1, 3, ..., 2047, 4095, ..., 65535)")
+	flags.Int(option.HubbleEventBufferCapacity, observeroption.Default.MaxFlows.AsInt(), "Capacity of Hubble events buffer. The provided value must be one less than an integer power of two and no larger than 65535 (ie: 1, 3, ..., 2047, 4095, ..., 65535)")
```
Comment (fristonio) [inline daemon/cmd/daemon_main.go:852]: We will need to generate cmdref again. `make -C Documentation update-cmdref`

**Review by fristonio — CHANGES_REQUESTED** [review]

**Review by fristonio — APPROVED** [review]
LGTM :rocket:

**Comment by fristonio** [comment]
No need to run the full CI.

**Review by christarazi — APPROVED** [review]
LGTM. Quick note on the release note: in general, we try to have a readable release note for our users. This PR will show up in the changelog as "Address nits" which is not useful to users or even developers. In this case, the change is quite small and low impact. Something along the lines of "Add more context to error msg ..." would suffice.  Because the original PR that this PR is updating hasn't made it into a release and this being low impact, this is fine to keep how it is. Merging.

**Pull request merged**